mkt/urls.py

```diff
@@ -63,6 +63,7 @@
 
     # Users.
     ('^users/', include(user_patterns)),
+    url('^users/session/$', user_session, name='users.session'),
```

Comment:
or in `user_patterns` ? (which come from `mkt.accounts.urls`, confusing... we should merge users/accounts apps)